src/components/VirtualizedAutocomplete/index.tsx

```diff
@@ -27,6 +29,8 @@ const ListboxComponent = ({
 	const itemData = (children as ItemDataElement[]).slice();
 	const { itemCount, loadNextPage } = props.pagination;
 	const [optionHeightsTotal, setOptionHeightsTotal] = React.useState(0);
+	const ref = React.useRef<VListHandle>(null);
+	const fetchedCountRef = React.useRef(-1);
```

Comment:
can't this be 0 ? i don't expect loadNextPage to load the first page as well, right ?

Reply:
I was just following their lead: https://github.com/aeharding/virtua/blob/ef1d10ba9a5483915d2ec743375c6b7563ff0ce7/stories/react/basics/VList.stories.tsx#L560

But looking closer, I think we don't need it at all since we have `itemData.length` to tell us how much we have already fetched